semantic-conventions/syntax.md

```diff
@@ -168,6 +178,26 @@ The following is only valid if `type` is `event`:
   If not specified, the `prefix` is used. If `prefix` is empty (or unspecified),
   `name` is required.
 
+#### Metric Group semantic convention
+
+Metric group inherits all from the base semantic convention, and does not 
+add any additional fields. Its only requirement is that `type` is `metric_group`. 
```

Comment:
```suggestion
add any additional fields.
```

I think this is so obvious that stating it explicitly is confusing.

Reply:
Done